packages/fx-core/src/common/spec-parser/specParser.ts

```diff
@@ -100,22 +167,37 @@ export class SpecParser {
     // TODO: implementation
   }
 
+  private getAllRemoteRefs(spec: any, root = ""): Map<string, string> {
```

Comment:
Looks like there is a paths() method (https://apitools.dev/swagger-parser/docs/swagger-parser.html#refs) provided by SwaggerParser. Just wondering if that would help so we don't have to handle this by ourselves.

Reply:
Thanks, I will take a look